src/inputs/DateFields/DateFieldBase.tsx

```diff
@@ -243,7 +251,23 @@ export function DateFieldBase(props: DateRangeFieldBaseProps | DateSingleFieldBa
   // TODO: figure this out... seems weird to have now that we support multiple dates formats....
   // How do other applications handle this defined sizing? Appears they use hard coded widths depending on format, which is similar here (using `size` instead of css `width`).
   // But would also need to allow for the input to be `fullWidth`, which is basically also what we're accomplishing here... so maybe fine?
-  const inputSize = mode !== "range" ? (format === "short" ? 8 : format === "medium" ? 10 : undefined) : undefined;
+  const inputSize = !isRangeMode ? (format === "short" ? 8 : format === "medium" ? 10 : undefined) : undefined;
+
+  // Support input range filter field w/ a clear btn that will appear when overlay is closed and input is not focused
+  const clearButton = (
+    <>
+      {(inputValue !== "" && !state.isOpen) && (
+        <IconButton
```

Comment:
**question(s)** Any reason to not utilize the `clearable` property on `TextFieldBase`? And, is there a specific reason why we do not want to show the "clear" button when the overlay is open. Doesn't seems like a bad thing to me to have that.

Reply:
Our `TextFieldBase` will only show the clear `X` if the field is focused where as we wanted it shown any time the field is filled. 
Only showing it when the datepicker is closed because I wasn't able to rework the overlayTrigger and `useEffect` on ln 209 to clear the field in **ONE** click. It was always requiring **TWO** clicks. Having it removed when closed does keep the user focused on the datePicker since it can be cleared by clicking the same date twice.